analyses/snv-callers/run_db_setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
```

Comment:
I'm going to presume I can eventually merge this with `run_caller_analysis.sh` so only one bash script needs to be run?

Reply:
That is the idea.